src/components/SingleArticle.jsx

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { fetchSingleArticle } from "../../api";
+
+export default function SingleArticle() {
+  const [currArticle, setCurrArticle] = useState({});
+  const { article_id } = useParams();
+
```

Comment:
Please add some loading feedback/state to let us know that something is happening, its just a slow server/Internet connection 💡